app/assets/stylesheets/dashboard.scss

```diff
@@ -257,45 +259,59 @@
   }
 }
 
-.charts-container {
-  max-width: 500px;
-  height: 250px;
-  margin: 0 auto;
-}
-
 .analytics-container {
   width: 1400px;
   margin: 0 auto;
 }
 
+.crayons-tabs--analytics {
+  justify-content: center;
+}
+
 .summary-stats {
+  text-align: center;
   display: flex;
-  flex-wrap: wrap;
-  width: 100%;
   justify-content: space-between;
 
-  .card {
-    flex: 1 0 150px;
+  &__col {
+    flex: 1;
+    width: 10%;
```

Comment:
I'm curious: how did we decide on `10%` width here, and `50%` and `90%` width below? 🤔 

I'm asking because it seems a little "magic number-y" to me, and I wonder if there is a reason why we picked those, or if they're just what looks good 😅 Either way, I think adding a comment and extracting them into a variable might be nice so it's clear to us down the road why we picked these styles if we ever want to go back and change them :)

Reply:
This was my mistake! I was fiddling with these values but ended up going with the `flex` route and those width percentages were not even taking any effect there. Thanks for catching this I just now cleaned these up 😄